app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -137,7 +137,7 @@ def store_request
     def pii_requested_but_locked?
       sp_session && sp_session_ial > 1 &&
         UserDecorator.new(current_user).identity_verified? &&
-        user_session[:decrypted_pii].blank?
+        Pii::Cacher.new(current_user, user_session).fetch.blank?
```

Comment:
for these `.fetch.blank?` and `.fetch.present?`, what if we added a method for that?

```suggestion
        Pii::Cacher.new(current_user, user_session).exists_in_session?
```

Reply:
That's a great idea, it would allow us to defer decryption a bit further even since it could check for whether the value exists in either decrypted/encrypted form

Reply:
I think in the future it would let us defer decryption, but I think by the time we're here, we've already decrypted the whole session payload?

Reply:
Added in ba75087d88e09f8e5b4c14b08e7dcc8f5d40ab32